src/sessions/session_inbound.cpp

```diff
@@ -35,8 +35,9 @@ using namespace bc::message;
 using namespace bc::network;
 using namespace std::placeholders;
 
-session_inbound::session_inbound(full_node& network, safe_chain& chain)
-  : session<network::session_inbound>(network, true),
+session_inbound::session_inbound(full_node& network, safe_chain& chain,
+    const bc::settings& bitcoin_settings)
```

Comment:
Node exposes settings so passing settings is redundant.

Reply:
Good catch! Done.